_pages/en_US/dsiware-backups.md

```diff
@@ -7,28 +7,31 @@ title: DSiWare Backups
 ## Requirements
 - The latest version of [GodMode9i](https://github.com/RocketRobz/godmode9i/releases)
    - Download the archive, extract the contents and place `GodMode9i.nds` anywhere on your SD card
-- A Nintendo DSi console with [Unlaunch](/unlaunch) installed
 
 ## Nintendo DSi - Instructions
 
-### Section I - Identifying the path to the .app file
-1. Boot your Nintendo DSi console while holding <kbd class="face">A</kbd> + <kbd class="face">B</kbd>
-   - This should take you to the Unlaunch Filemenu
-1. Identify your application in the list
-   - A light green entry means that they are installed to NAND
-
-When you hover over the entry, the bottom screen should show the path. Take note of the path.
-
-An example path would be `nand:/title/0030005/484e4a45/content/00000003.app`
-{: .notice--info}
-
-### Section II - Extracting the file
-1. Launch GodMode9i from the Unlaunch Filemenu
-1. Navigate to SYSNAND, then navigate through the path you took note of above
+### Section I - Identifying the desired DSiWare
+1. Launch GodMode9i and select `[nand:] SYSNAND`
+1. Navigate to the `title` folder
+1. Choose the folder according to whichever category you're looking for
+   - `00030004`: Standard DSiWare
+   - `00030005`: Pre-installed Fun Tools
+   - `00030015`: System Base Tools
+1. Once you have chosen which type of DSiWare you would like to extract, enter a subfolder, and then enter `content`
+1. There should now be an `.app` file visible. Select the file, and choose `Show NDS file info`. This will tell you if it's the DSiWare that you are looking for
+   - If it is not the DSiWare title that you were looking for, continue searching in other folders until you find it

```

Comment:
Might want to mention that if the file doesn't have a banner / isn't a real NDS file, which can happen for some like the `0003000f` files.